app/api/metrics/route.ts

```diff
@@ -47,7 +47,9 @@ const createLogPairs = (ackLogs: Array<ethers.EventLog>, sendPacketLogs: Array<e
 
     if (ackLogsMap.has(key)) {
       const ackLog = ackLogsMap.get(key)!;
-      logPairs.push({ackLog, sendPacketLog});
+      if (ackLog.blockNumber > sendPacketLog.blockNumber) {
+        logPairs.push({ackLog, sendPacketLog});
+      }
```

Comment:
Bug fix for when a ack packet event has a timestamp earlier than send packet event